docs/src/systems/boundary.md

```diff
@@ -93,7 +97,20 @@ where the sum is over all fluid particles, ``\rho_f`` and ``p_f`` denote the den
     AdamiPressureExtrapolation
 ```
 
-#### 4. [`PressureZeroing`](@ref)
+#### 2. [`BernoulliPressureExtrapolation`](@ref)
+Identical to [`AdamiPressureExtrapolation`](@ref), but it adds the dynamic pressure component of the Bernoulli equation:
+```math
+\frac{1}{2} \, \rho_{\text{neighbor}} \left( \frac{ (\mathbf{v}_f - \mathbf{v}_{\text{neighbor}}) \cdot (\mathbf{x}_f - \mathbf{x}_{\text{neighbor}}) }{ \left\| \mathbf{x}_f - \mathbf{x}_{\text{neighbor}} \right\| } \right)^2 \times \text{factor}
+```
+where ``v_f`` is the velocity of the fluid and ``v_\text{body}`` is the velocity of the body.
```

Comment:
Please update this with the new symbols used in the equation.

Reply:
This is only unclear because this is only called for a moving body so relative_velocity is v_body-v_fluid_neighbor which is the relative velocity of the fluid to the body movement.